core/2d/DrawNode.h

```diff
@@ -79,7 +74,7 @@ class AX_DLL DrawNode : public ax::Node
     {
         Circle,
         Rect,
-        Triangle,
+        Triangle, // not implemented at the moment
```

Comment:
If it's not implemented, then it should be removed.

Reply:
good idea. removed